battle-engine.js

```diff
@@ -2045,6 +1955,8 @@ Battle = (() => {
 		this.messageLog = [];
 
 		this.startingSeed = this.generateSeed();
+
+		this.send = send;
```

Comment:
In theory you just need to have it call `PM.send` instead of passing around a callback.